libs/wingc/src/jsify.rs

```diff
@@ -102,6 +107,7 @@ impl JSifier {
 
 		if self.shim {
 			output.push(format!("const {} = require('{}');", STDLIB, STDLIB_MODULE));
+			output.push(format!("const core = $stdlib.core;"));
```

Comment:
```suggestion
			output.push(format!("const $core = $stdlib.core;"));
```

As a convention, maybe prefix with $ symbols that are owned by the compiler

Reply:
done